src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/SeparatorWrapCheck.java

```diff
@@ -220,18 +222,18 @@ public void visitToken(DetailAST ast) {
         final String text = ast.getText();
         final int colNo = ast.getColumnNo();
         final int lineNo = ast.getLineNo();
-        final String currentLine = getLines()[lineNo - 1];
-        final String substringAfterToken =
-                currentLine.substring(colNo + text.length()).trim();
-        final String substringBeforeToken =
-                currentLine.substring(0, colNo).trim();
+        final int[] currentLine = getLineCodePoints(lineNo - 1);
+        final int[] substringAfterToken = CodePointUtil.trim(Arrays.copyOfRange(currentLine,
+                colNo + text.length(), currentLine.length));
```

Comment:
Please do:

```
        final int[] substringAfterToken = CodePointUtil.trim(
                Arrays.copyOfRange(currentLine, colNo + text.length(), currentLine.length)
        );

```

Or extract `Arrays.copyOfRange...` to separate variable.

Reply:
done

Reply:
Resolved.